src/store/substrate/index.js

```diff
@@ -173,7 +175,7 @@ export default {
           console.log("Is pair locked?", pair.isLocked)
           commit("SET_WALLET", pair)
           
-          localStorage.setLocalStorageByName("mnemonic_data", CryptoJS.AES.encrypt(file[1].mnemonic, password));	
```

Comment:
this line supposed to be the correct one

Reply:
i checked this code and compare with development branch, and i change this code base on development branch